steps/set-env-var/0.9.1/step.yml

```diff
@@ -0,0 +1,46 @@
+title: Set environment variable
+summary: |
+  Set environment variable
+description: |
+  With this step you can copy a value into as many environment variables as you like.
```

Comment:
I think this would be better:
With this step, you can copy a value into environment variables.